node/src/effect.rs

```diff
@@ -76,7 +77,9 @@ use once_cell::sync::Lazy;
 use serde::{Deserialize, Serialize};
 use smallvec::{smallvec, SmallVec};
 use tokio::{sync::Semaphore, time};
-use tracing::error;
+#[cfg(not(feature = "fast-sync"))]
```

Comment:
This should be removed.

Reply:
Fixed in https://github.com/casper-network/casper-node/pull/2417/commits/2b7c5ef9fdba9ab09d0587bfba42c5b9be0353a0